pkg/controllers/flannelmigration/migration_controller.go

```diff
@@ -383,8 +383,8 @@ func (c *flannelMigrationController) runIpamMigrationForNodes() ([]*v1.Node, err
 
 	// Work out list of nodes not running Calico. It could happen that all nodes are running Calico and it returns an empty list.
 	items := c.indexer.List()
-	var controllerNode *v1.Node
-	var masterNode *v1.Node
+	var controllerNodes []*v1.Node
```

Comment:
Not sure if we need this change since there should always be a single node running migration controller.

Reply:
Yeah, true. This is only ever going to be one item long.

Reply:
I will leave it like this though, just in case something weird happens we'll handle it more gracefully.